### PR TITLE
Port over the vue-3 cypress fix branch

### PR DESCRIPTION
**Pull request by roblevintennis**
Ports over changes from https://github.com/morkro/vue-a11y-dialog/pull/35 to `1.1.0` branch per request

### CODE REVIEW

**Review by morkro — COMMENTED** [review]
Looks good @roblevintennis :) Just a few comments on consistency and clarity.

I also noticed that a lot of re-formatting (e.g. semi-colons, format) has happened. Seems like your IDE picks up a different linting or the local setup is not working? Either way, can be ignored for this PR.

Additionally, what's the `package_test/` directory about? Don't think this is needed for this repository as we already have demo files. Let's keep the changes relevant to the bug fix only.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-        uses: actions/checkout@v2
-
-      - name: Install npm dependencies
-        run: npm ci
-
-      - name: Run lint
-        run: npm run lint
-
-      - name: Run tests
-        run: npm test
```
[the remark this answers is not in the document]
Comment (roblevintennis) [reply .github/workflows/ci.yml:23]: Yeah, bad port good catch

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
[the remark this answers is not in the document]
Comment (roblevintennis) [reply cypress/fixtures/example.json:1]: Nope

**Comment by roblevintennis** [comment]
Looks like this is all getting cherry-picked over into `1.1.0` branch so I'll hold off on any updates since that's quite close I think. Feel free to close this PR if you're done referencing it.

**Comment by morkro** [comment]
Thanks @roblevintennis! :) Will fix the tests in #33 and then we should be ready to release

**Comment by roblevintennis** [comment]
@morkro I left some comments on the PR and figured out how to achieve your great idea to pull in the esm build in demo. I'm sure you're seeing on PR but just in case: https://github.com/AgnosticUI/vue-a11y-dialog/commit/93daea064e6e71a9951e78d43de9cc8149c9fbfd gets tests green and also allows for using the demo as a vue app directly with `npm run dev`.

**Pull request closed without merging**